live-editing/configs/GridConfigGenerator.ts

```diff
@@ -21,17 +10,29 @@ import {
     IgxProgressBarModule,
     IgxRippleModule,
     IgxSwitchModule,
-    IgxInputDirective,
     IgxToastModule
 } from "igniteui-angular/main";
-import { DataService } from "../../src/app/grid/services/data.service";
-import { LocalService } from "../../src/app/grid/grid-sample-2/grid-sample-2.component";
-import { GridContextmenuComponent } from "../../src/app/grid/grid-sorting-sample/grid-contextmenu/grid-contextmenu.component";
+import { FilteringSampleComponent } from "../../src/app/grid/grid-filtering-sample/grid-filtering-sample.component";
+import { PagingSampleComponent } from "../../src/app/grid/grid-paging-sample/grid-paging-sample.component";
+import { ResizingSampleComponent } from "../../src/app/grid/grid-resizing-sample/grid-resizing-sample.component";
+import { FinancialSampleComponent, LocalService } from "../../src/app/grid/grid-sample-2/grid-sample-2.component";
```

Comment:
The topic is perfect! Well done Tishe.

One thing that I've noticed is that Grid column resizing stackblitz stample is not loaded correctly.

I don't know why, but this sample is importing FinancialSampleComponent


![image](https://user-images.githubusercontent.com/11193764/39001580-4a84eb54-43ff-11e8-803a-e845f934fb4a.png)